src/app/index.ts

```diff
@@ -98,6 +102,14 @@ if (app.get('env') === 'production') {
 
 app.use(cookieParser());
 app.use(session(sess));
+// apply other headers to reponse
+app.use((_, res, next) => {
```

Comment:
I think this is the recommended way to pass this header? https://github.com/helmetjs/helmet/issues/351#issuecomment-1015498560

Reply:
@machariamuguku I do not think csp helmet currently supports report-to header.